lib/deobfuscate.py

```diff
@@ -7,6 +10,20 @@
 DEFAULT_MAX_DEPTH = 150
 
 
+def parse_base64_payload(payload: str) -> str:
+    if -1 == payload.find(r"Command/Base64/"):
```

Comment:
This code pattern reduces readability, please consider:
```suggestion
    if "Command/Base64/" not in payload:
```

Reply:
Fixed